src/index.js

```diff
@@ -10,11 +10,16 @@ export default function (contents) {
     external: RegExp().test.bind(/^(.(?!\.(js|es)$))*$/)
   }
 
-  const plugins = this.options.rollup || loaderUtils.parseQuery(this.query)
-  const options = Array.isArray(plugins) ? { plugins } : plugins
+  const config = loaderUtils.getLoaderConfig(this, 'rollup')
+  const options = this.options.rollup
```

Comment:
It's still needed as the `this.options.rollup` when an array and processed by `getLoaderConfig` is converted into an object of index keys. Which is why I do the little bit of extra processing to normalize it.